kimchi/src/verifier.rs

```diff
@@ -539,6 +539,13 @@ where
                                 scalars.push(scalar * j);
                                 commitments.push(t);
                             }
+                            if let Some(table_ids) = lindex.table_ids.as_ref() {
+                                scalars.push(
+                                    (scalar * constants.joint_combiner)
+                                        .pow([lindex.max_joint_size as u64]),
```

Comment:
shouldn't this be 

```rust
scalar * (constants.joint_combiner.pow(...))
```

Reply:
Good catch!

Reply:
scalar is probably `1` anyway here right?

Reply:
Must be, because my tests passed. Still, better to fix it for future :)